completions_v2.ts

```diff
@@ -0,0 +1,198 @@
+// Copyright 2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
Is this module new code? I can't tell where it came from?

Reply:
This is https://github.com/denoland/dotland/blob/main/worker/suggestions.ts, but went through two refactors